### ISSUE TITLE
Predicted Effect

### ISSUE THREAD

**Issue by mumusanren** [ETSSmartRes/VAE-NILM#2]
Hello, thank you for your efforts on Nilm research. Your published papers and vae_nilm project are very helpful for my research.I tried to train the Nilm model with your project. The data is uk_dale low frequency data of washing machine with epoch 15 (I see that each epoch takes about 20 minutes and there are large fluctuations at about 12 times). The final training result is not as described in your article, please tell me if I have a problem in parameter selection.
![WashingMachine1](https://user-images.githubusercontent.com/83314953/139818154-ebebd37d-a3a7-4686-9f9b-5a212701c367.png)
![1635844583(1)](https://user-images.githubusercontent.com/83314953/139819092-96b22825-c08b-415a-b387-0c14d18505e4.jpg)

**Comment by ETSSmartRes** [comment]
Hello,
At first, I would say maybe you have a problem with the data. The shape of your True data looks weird. Did you use the 
```
python uk_dale_preprocess.py
```
to generate your data? And which houses are included in your training and test data. 

Second, if you use Google Colab with GPU, each epoch should take around 180 seconds.

Finally, here is an example of learning curves for the loss, vall_loss, Mean_absolute_error and val_Mean_absolute_error for training on houses 1 and 5, and test on house 2 for the WashingMachine appliance.

![image](https://user-images.githubusercontent.com/56851318/139846279-3418dac8-5cb0-477a-9544-3df82933952a.png)
![image](https://user-images.githubusercontent.com/56851318/139846359-454ce18d-b339-41cc-b61e-727077148d06.png)
![image](https://user-images.githubusercontent.com/56851318/139846386-8f819577-e5ea-459a-982e-ac7061d93bcc.png)
![image](https://user-images.githubusercontent.com/56851318/139846414-85c0dd77-656a-42bd-ab65-802910c1286d.png)

```
[[5.98107709]
 [0.82262452]
 [0.84957903]
 [0.94867943]
 [0.89639856]
 [0.35475097]]
```

Did you use all the same hyperparameters as the ones proposed in the paper?

Thanks,

Antoine

**Comment by mumusanren** [comment]
Thank you for your answer. I fixed the cuda and cudnn installed on my computer and the training speed improved a lot. Retrained the model and now it is performing well.
